### DISCUSSION TITLE
Arbitrary polling for admins

### DISCUSSION THREAD

**Discussion by craftxbox** [unitystation/unitystation#7110] [Ideas]
### Description
Allow admins to run polls similar to the restart vote with custom text and options

**Comment by corp-0** [comment]
would you mind if I make this a discussion instead? So the idea can mature and be more explicit on how it should work and what's needed to consider the issue solved.

**Comment by craftxbox** [comment]
Sure, go ahead.

**Comment by MaxIsJoe** [comment]
> ### Description
> Allow admins to run polls similar to the restart vote with custom text and options

I'm already on that along side the round start tips, just give me time though.

**Comment by Fogapod** [comment]
Displaying poll for specific players (all antags/all nonantags/manually selected players) might be useful for large pop events

**Comment by MaxIsJoe** [comment]
Update on this, the base functionality to let this happen is in.
Just add the ability for admins to start their own polls now.
